substrate/client/service/test/src/client/mod.rs

```diff
@@ -1748,11 +1748,9 @@ fn respects_block_rules() {
 }
 
 #[test]
-#[cfg(disable_flaky)]
 #[allow(dead_code)]
-// FIXME: https://github.com/paritytech/substrate/issues/11321
 fn returns_status_for_pruned_blocks() {
```

Comment:
The test is not flaky anymore? :D

Reply:
🙈 we have an auto-retry in the CI now, i will keep the FIXME comment.